docs/learn.md

```diff
@@ -400,7 +400,318 @@ over the way these JavaScript packages are downloaded and bundled.
+│   └── lib.ml
+├── dune-project
+├── dune
+└── main.ml
+```
+
+#### Building the project
+
+We can build the project now, which will produce the JavaScript code from our
+sources using the Melange compiler:
+
+```bash
+$ dune build @melange
+```
+
+This command tells dune to build all the targets that have an alias `melange`
+attached to them.
+[Aliases](https://dune.readthedocs.io/en/stable/overview.html#term-alias) are
+build targets that don’t produce any file and have configurable dependencies.
+
+By default, all the targets in a `melange.emit` stanza and the libraries it
+depends on are attached to the `melange` alias. We can define explicit aliases
+though, as we will see below.
+
+If everything went well, we should be able to run the resulting JavaScript with
+Node. As we mentioned while going through its features, Dune places all
+artifacts inside the `_build` folder to not pollute any source folders. So we
+will point Node to the script placed in that folder, to see the expected output:
+
+```bash
+$ node _build/default/app/app.js
+Jane
+```
+
+One thing to note is that we have to look for the `app.js` file inside an `app`
+folder, but we don’t have any such folder in our sources. The reason why this
+folder is created is to support multiple `melange.emit` stanzas in the same
+folder. To support this scenario, Dune will use the `target` field defined in
+the `melange.emit` to place the artifacts generated from a `melange.emit` stanza
+in the following folder:
+
+```
+_build/default/$path-to-directory-of-melange-emit-stanza/$target
+```
+
+This allows to have two or more `melange.emit` stanzas in the same folder
+without conflicts or overrides between each other.
+
+#### Using aliases
+
+The default `melange` alias is useful for prototyping or when working on small
+projects, but larger projects might define multiple entry points or
+`melange.emit` stanzas. In these cases, it is useful to have a way to build
+individual stanzas. To do so, one can define explicit aliases for each one of
+them by using the `alias` field.
+
+Let’s define a custom alias `app` for our `melange.emit` stanza: 
+
+```bash
+(melange.emit
+ (target app)
+ (alias app)
+ (libraries lib))
+```
+
+Now, when building with Dune, we can refer to this new alias:
+
+```bash
+$ dune build @app
+```
+
+Note that if we try to build again using the default `melange` alias, Dune will
+return an error, as there are no more targets attached to it.
+
+```bash
+$ dune build @melange
+Error: Alias "melange" specified on the command line is empty.
+It is not defined in . or any of its descendants.
+```
+
+#### Handling assets
+
+The last topic we will go through in this demo project is asset handling.
+Sometimes we want to use CSS files, fonts, or other assets in our Melange
+projects. Due to the way Dune works, our assets will have to be copied to the
+`_build` folder as well. To make this process as easy as possible, the Melange
+integration with Dune provides two ways to do this:
```

Comment:
it's just one way. it's confusing if we mention it's 2 ways. It's the same mechanism that works in the supported stanzas.

Reply:
reworded in https://github.com/melange-re/documentation-site/pull/25/commits/d2d429b270fbc284f929b8d292410c531dbfc6d0.